### ISSUE TITLE
Idd object list error for BiquadraticCurves in VRF related objects and other case inconsistencies

### ISSUE THREAD

**Issue by kbenne** [NREL/EnergyPlus#4856]
In AirConditioner:VariableRefrigerantFlow and maybe others we use \object-list BiQuadraticCurves, however the reference is actually BiquadraticCurves with a lower "q".  This makes a difference to at least OpenStudio's IDD parser and object list tracking.

```
  A3 , \field Cooling Capacity Ratio Modifier Function of Low Temperature Curve Name
       \type object-list
       \object-list BiQuadraticCurves
```

**Comment by mjwitte** [comment]
A quick search on the IDD finds 27 instances of BiQuadraticCurves.  Need to check all \object-list to be sure they match case with a \reference .

**Comment by mjwitte** [comment]
Closed via #5103
